R/tmb.R

```diff
@@ -171,7 +171,9 @@ h_mmrm_tmb_data <- function(formula_parts,
     toep = 2,
     ar1 = 3,
     ar1h = 4,
-    ad = 5
+    ad = 5,
+    cs = 6,
+    csh = 7
```

Comment:
Still digesting the code base, but this stood out to me as a curious design choice. If I'm understanding this correctly, this maps the covariance types to their enum counterparts, but I would expect this to happen closer to wherever the C interface is called instead of losing the information at this point in the processing. As it is now, it would be rather difficult to recover this information.

Reply:
Good catch! I know that this is not pretty. Thanks for commenting on it, I will make a separate issue for this as it will require some medium refactoring.

Reply:
https://github.com/openpharma/mmrm/issues/69